.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
this shouldn't be pushed

Reply:
For sure! Don't know how it was possible...